docs/Maintainer-Guidelines.md

```diff
@@ -104,3 +104,14 @@ Whitespace corrections (to Ruby standard etc.) are allowed (in fact this
 is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
+
+## Communication
+Maintainers have a variety of ways to communicate with each other:
+
+- Homebrew's public repositories on GitHub
+- Homebrew's group communications between more than two maintainers on private channels (e.g. GitHub/Slack/Discourse)
+- Homebrew's direct 1:1 messages between two maintainers on private channels (e.g. iMessage/Slack/Discourse/IRC/carrier pigeon)
+
+All communication should ideally occur in public on GitHub. Where this is not possible (e.g. a security disclosure, interpersonal issue between two maintainers, urgent breakage that needs to be resolved) this can move to maintainers' private group communication and, if necessary, 1:1 communication. Technical decisions should not happen in 1:1 communications but if they do (or did in the past) they must end up back as something linkable on GitHub. For example, if a technical decision was made a year ago on Slack and another maintainer/contributor/user asks about it on GitHub, that's a good chance to explain it to them and have something that can be linked to in the future.
```

Comment:
I'd suggest adding something like 'or practical' after:
> Where this is not possible

Reply:
@maxim-belkin good call, updated.

Reply:
But when is something “not practical”? You could pretty much argue that about anything.

Reply:
my comment was targeted at 'security issues' and 'urgent breakage': it is not practical to discuss them in public, meaning, it does not make a lot of sense to do that.

The same somewhat applies to interpersonal issues, though I wouldn't call them out here.

Reply:
Can we change it to “appropriate”? This meaning of “practical” isn't the first thing I would think of as a non-native speaker.

Reply:
@reitermarkus Agreed, pushed.

Reply:
`appropriate` is appropriate